templates/admin/stats/github.tx

```diff
@@ -0,0 +1,108 @@
+<style>
+    /* typography */
+    h2 {
+        font-size: 1rem;
+    }
+    .number {
+        font-size   : 2rem;
+        font-weight : bold;
+    }
+    #last_updated {
+        font-size : .8rem;
+    }
+    a {
+        text-decoration : none;
+    }
+
+    /* layout */
+    h1 {
+        margin-bottom  : 2;
+        padding-bottom : 0;
+    }
+    h2 {
+        margin-bottom  : 0;
+        padding-bottom : 0;
+    }
+    .data {
+        height : 4rem;
+    }
+    #last_updated {
+        margin-top    : 0;
+        padding-top   : 0;
+        margin-bottom : 2rem;
+        text-align    : center;
+    }
+    .tabs {
+        border-bottom : 1px solid #e8e8e8;
+        margin-top    : 1rem;
+        margin-bottom : 1rem;
+        font-size     : .8rem;
+        font-weight   : bold;
+        color         : #999;
+    }
+    .tabs .tab {
+        display : inline-block;
+        margin  : 0 0 -1px;
+    }
+    .tabs .tab a {
+        color   : #999;
+        display : block;
+        padding : 8px 15px 9px;
+    }
+    .tabs .tab.selected {
+        color         : #333;
+        border        : 1px solid #e8e8e8;
+        border-bottom : none;
+        border-radius : 3px 3px 0 0;
+        background    : #fff;
+    }
+    .space {
+        height : 2rem;
+    }
+
+
+    /* colors */
+    h2 {
+        color : #aaa;
+    }
+
+    /* state */
+    .loader {
+        visibility : hidden;
+    }
```

Comment:
@kablamo You could use the "hide" class instead. Anyway yeah please export the style to an scss file inside src/scss/ia/, and import it in [src/scss/ia/main.scss](https://github.com/duckduckgo/community-platform/blob/master/src/scss/ia/main.scss) - also, instead of the hex colours, you can use [variables](https://github.com/duckduckgo/duckduckgo-colors/blob/master/_colors.scss) :)